File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();
        double[] numArr = new double[num];

        double maxVal = 0.0;
        double sum = 0.0;

        for (int i = 0; i < num; i++) {
            double temp = sc.nextInt();
            numArr[i] = temp;
            maxVal = Math.max(maxVal, temp);
        }

        for (double v : numArr) {
            sum += v / maxVal * 100;
        }

        System.out.println(sum/num);

    }
}

File: 백준/Bronze/1546. 평균/평균.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    var maxVal = 0.0

    val num = sc.nextInt()
    val numArr = DoubleArray(num)

    var sum = 0.0

    for (i in 0 until num) {
        val temp = sc.nextInt().toDouble()
        numArr[i] = temp
        maxVal = maxVal.coerceAtLeast(temp)
    }

    numArr.forEach { sum += it/maxVal*100 }

    println(sum/num.toDouble())

}